test/functions_test.rb

```diff
@@ -121,6 +120,18 @@ def test_function_with_error
       assert_equal "[SassC::FunctionsHandler] Intentional wrong thing happened somewhere inside the custom function", stderr_output
     end
 
+    def test_function_that_returns_a_sass_value
```

Comment:
What about this:

```ruby
def test_function_that_returns_a_sass_value
  assert_sass <<-SASS, <<-CSS
    div { background: returns-sass-value(); }
  SASS
    div { background: black; }
  CSS
end

# ...

private

def assert_sass(sass_template, expected_css)
  expected = expected_css.squish
  actual = Engine.new(sass_template).render.squish
  assert_equal expected, actual
end
```

(Beyond looking cleaner, this also enables CSS highlighting in editors like Sublime.)